Src/IronPython/Modules/unicodedata.cs

```diff
@@ -397,17 +397,17 @@ internal static class StreamReaderExtensions {
 
     class CharInfo {
         internal CharInfo(string[] info) {
-            this.Name = info[PropertyIndex.Name].ToUpperInvariant();
-            this.General_Category = info[PropertyIndex.General_Category];
-            this.Canonical_Combining_Class = int.Parse(info[PropertyIndex.Canonical_Combining_Class]);
-            this.Bidi_Class = info[PropertyIndex.Bidi_Class];
-            this.Decomposition_Type = info[PropertyIndex.Decomposition_Type];
+            Name = info[PropertyIndex.Name].ToUpperInvariant();
+            General_Category = info[PropertyIndex.General_Category];
+            Canonical_Combining_Class = int.Parse(info[PropertyIndex.Canonical_Combining_Class]);
+            Bidi_Class = info[PropertyIndex.Bidi_Class];
+            Decomposition_Type = info[PropertyIndex.Decomposition_Type];
 
             string nvdes = info[PropertyIndex.Numeric_Value_Decimal];
-            this.Numeric_Value_Decimal = nvdes != "" ? (int?)int.Parse(nvdes) : null;
+            Numeric_Value_Decimal = nvdes != "" ? (int?)int.Parse(nvdes) : null;
```

Comment:
Can this use ?? instead?

Reply:
We are checking if the string is empty in this case, not null. Or you mean something else?

Reply:
No, I was reading it at 3am and misread it :)